<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QMakeBuildDirChooser</class>
 <widget class="QWidget" name="QMakeBuildDirChooser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>611</width>
    <height>200</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>200</height>
   </size>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::ExpandingFieldsGrow</enum>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>QMake &amp;executable:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>kcfg_qmakeExecutable</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="KUrlRequester" name="kcfg_qmakeExecutable">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>&amp;Build directory:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>kcfg_buildDir</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="KUrlRequester" name="kcfg_buildDir"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>&amp;Install to (target.path):</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>kcfg_installPrefix</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="KUrlRequester" name="kcfg_installPrefix">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Build &amp;type:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>kcfg_buildType</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QComboBox" name="kcfg_buildType">
     <property name="editable">
      <bool>false</bool>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <item>
      <property name="text">
       <string notr="true">Debug</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">Release</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>(Use default from .pro file)</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Extra arguments:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="KLineEdit" name="kcfg_extraArgs"/>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="KMessageWidget" name="status" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QWidget</extends>
   <header>kurlrequester.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KMessageWidget</class>
   <extends>QFrame</extends>
   <header location="global">kmessagewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>kcfg_qmakeExecutable</tabstop>
  <tabstop>kcfg_buildDir</tabstop>
  <tabstop>kcfg_installPrefix</tabstop>
  <tabstop>kcfg_buildType</tabstop>
  <tabstop>kcfg_extraArgs</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
